src/crimson/os/seastore/seastore.h

```diff
@@ -310,8 +327,35 @@ class SeaStore final : public FuturizedStore {
     std::map<std::string, ceph::bufferlist>&& kvs);
 
   boost::intrusive_ptr<SeastoreCollection> _get_collection(const coll_t& cid);
+
+  static constexpr auto LAT_MAX = static_cast<std::size_t>(lat_t::MAX);
+  struct count_lat_t {
+    int64_t num = 0;
+    std::chrono::steady_clock::duration duration_ms =
+                               std::chrono::steady_clock::duration::zero();
+  };
+  struct {
+    std::array<count_lat_t, LAT_MAX> seastore_op_lat;
+  } stats;
+
+  count_lat_t& get_latency(
+      std::array<count_lat_t, LAT_MAX>& seastore_op_lat,
```

Comment:
why do we need to pass `seastore_op_lat` to this function? i think we can just hardwire it to `stats.seastore_op_lat` right in the implementation of this function.